From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Mon, 17 Jan 2022 07:30:38 +0100
Subject: Testing: Ignore test around git tooling

We build based on extracted data from an tarball, therefore the git
checking functions can't work and we place simply a static link to the
upstream data.
---
 src/twisted/python/test/test_release.py | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/twisted/python/test/test_release.py b/src/twisted/python/test/test_release.py
index d55a40e..78d0a5f 100644
--- a/src/twisted/python/test/test_release.py
+++ b/src/twisted/python/test/test_release.py
@@ -55,6 +55,8 @@ GENERIC_CI_ENVIRON = {
 }
 
 
+skip = "Not relevant within Debian."
+
 class ExternalTempdirTestCase(TestCase):
     """
     A test case which has mkdir make directories outside of the usual spot, so
@@ -514,6 +516,8 @@ class CommandsTestMixin(StructureAssertingMixin):
     Test mixin for the VCS commands used by the release scripts.
     """
 
+    skip = "Not relevant within Debian."
+
     def setUp(self):
         self.tmpDir = FilePath(self.mktemp())
 
@@ -599,6 +603,8 @@ class GitCommandTest(CommandsTestMixin, ExternalTempdirTestCase):
     L{GitCommand}.
     """
 
+    skip = "Not relevant within Debian."
+
     createCommand = GitCommand
 
     def makeRepository(self, root):
@@ -669,6 +675,8 @@ class CheckNewsfragmentScriptTests(ExternalTempdirTestCase):
     L{CheckNewsfragmentScript}.
     """
 
+    skip = "Not relevant within Debian."
+
     def setUp(self):
         self.origin = FilePath(self.mktemp())
         _gitInit(self.origin)
